test/integration/tests/metagov.ts

```diff
@@ -0,0 +1,457 @@
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { solidity } from 'ethereum-waffle';
+import { ethers } from 'hardhat';
+import { NamedContracts } from '@custom-types/types';
+import proposals from '@test/integration/proposals_config';
+import { TestEndtoEndCoordinator } from '@test/integration/setup';
+import { getImpersonatedSigner, expectRevert, time } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+import { MockOZGovernorVoter } from '@custom-types/contracts';
+import { deploy } from '@scripts/deploy/compoundPCVDeposit';
+
+const e18 = (x) => ethers.constants.WeiPerEther.mul(x);
+
+describe('e2e-metagov', function () {
+  let deployAddress: string;
+  let contracts: NamedContracts;
+  let e2eCoord: TestEndtoEndCoordinator;
+  const logging = Boolean(process.env.LOGGING);
+
+  before(async () => {
+    chai.use(CBN(ethers.BigNumber));
+    chai.use(solidity);
+  });
+
+  before(async function () {
+    deployAddress = (await ethers.getSigners())[0].address;
+    const config = {
+      logging,
+      deployAddress,
+      version: 1
+    };
+    e2eCoord = new TestEndtoEndCoordinator(config, proposals);
+    ({ contracts } = await e2eCoord.loadEnvironment());
+  });
+
+  describe('utils/VoteEscrowTokenManager.sol', function () {
+    let locker: any;
```

Comment:
`locker` should have a specific contract type rather than `any`

Reply:
I'll type